hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/RecordStore.java

```diff
@@ -324,4 +322,11 @@
     Storage getStorage();
 
     boolean isEvictionEnabled();
+
+    /**
+     * Starts mapLoader
+     */
+    void startLoading();
+
+    void onPut(Data key, Object value);
```

Comment:
Can you add some documentation? What is the purpose; does it trigger e.g. an event? When does this need to be called?